dashboard/dashboard.py

```diff
@@ -20,6 +21,9 @@
     }
 ]
 
+df = pandas.read_csv('https://raw.githubusercontent.com/plotly/datasets/master/solar.csv')
```

Comment:
I guess that this was from some test and can be removed. We still don't deal with solar energy, maybe in the future.